DeusEx/Classes/DXREnemies.uc

```diff
@@ -4,6 +4,20 @@ function FirstEntry()
 {
     Super.FirstEntry();
     RandoEnemies(dxr.flags.enemiesrandomized);
+    if( dxr.localURL == "INTRO" || dxr.localURL == "ENDGAME1" || dxr.localURL == "ENDGAME2" || dxr.localURL == "ENDGAME3" || dxr.localURL == "ENDGAME4" || dxr.localURL == "00_TRAINING" )
+    { // extra randomization in the intro for the lolz, ENDGAME4 doesn't have a DeusExLevelInfo object though, so it doesn't get randomized :(
+        RandomizeIntro();
```

Comment:
maybe RandomizeIntro should be a standard module function, so DXRando calls it on every module after AnyEntry()